.github/workflows/integration-sgx.yaml

```diff
@@ -0,0 +1,22 @@
+name: Cloud Hypervisor Tests (SGX)
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  build:
+    if: github.event_name == 'pull_request'
+    name: Tests (SGX)
+    runs-on: jammy-sgx
+    steps:
+      - name: Code checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - name: Run SGX integration tests
+        timeout-minutes: 60
+        run: scripts/dev_cli.sh tests --integration-sgx
+      - name: Run SGX integration tests for musl
+        timeout-minutes: 60
```

Comment:
Last time this ran the whole job took less than 10 minutes. I would set each of these to 10 minutes - we want to know if it's going to fail earlier.

Reply:
I picked this value from Jenkinsfile